spec/jobs/participant_outcomes/batch_send_latest_outcomes_job_spec.rb

```diff
@@ -44,17 +29,17 @@
     context "when there are more than batch_size records" do
       let(:batch_size) { 1 }
 
-      it "requeues itself" do
-        expect { described_class.perform_now(batch_size) }.to have_enqueued_job(described_class).with(batch_size, default_delay)
-      end
-
       it "only enqueues the send job for the first records up to the batch_size" do
         described_class.perform_now(batch_size)
 
         expect(ParticipantOutcomes::SendToQualifiedTeachersApiJob).to have_been_enqueued.exactly(:once).with(participant_outcome_id: 1)
         expect(ParticipantOutcomes::SendToQualifiedTeachersApiJob).not_to have_been_enqueued.with(participant_outcome_id: 2)
       end
     end
+
+    it "does not requeue itself" do
+      expect { described_class.perform_now(default_batch_size) }.not_to have_enqueued_job(described_class)
```

Comment:
I don't think this is testing the retry behaviour; it's calling `perform` and checking it doesn't enqueue itself - which it doesn't - but it doesn't check if Sidekiq requeues it on failure?

As the retry behaviour is an implementation detail of Sidekiq we probably don't need to test it 🤔 although asserting the `sidekiq_options` are correct may be worthwhile, but I'm not sure how straight-forward that would be. There is [a gem](https://github.com/philostler/rspec-sidekiq#be_retryable) that can do it, but it may break our other job-related tests.

Reply:
Agree around not testing retry behaviour from sidekiq, also since the implementation of manually requeuing in the job itself has been removed I don't think we need to test that it doesn't either